components/Hero/Wave.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import './wave.module.css'
+
+export const Wave = () => {
+  return (
+    <div className='wave'>
+        <div className="header__wave-desktop">
```

Comment:
because you have put in wave in its own component... I would name the class.

wave__content--desktop